agenda-webapps/src/main/webapp/vue-app/agenda-user-setting/components/AgendaUserGeneralSettings.vue

```diff
@@ -33,8 +35,10 @@
                     class="ma-2"
                     color="primary">
                     <template v-if="reminder.before">
-                      {{ $t('agenda.label.notifyMeBefore', {0: reminder.before, 1: $t(`agenda.option.${reminder.beforePeriodType.toLowerCase()}s`).toLowerCase()}) }}
-                    </template>
+                      <span class="{ mobile-chip-ellipsis : isMobile }">
```

Comment:
No need to add a new class here, please use existing CSS Class helpers (`text-truncate`) to apply ellipsis on elements (independently from mobile or desktop view). For each specific HTML element, you can add a dedicated CSS Class to add `max-width` property to fix the maximum allowed width. Sometimes, you don't even need to add an exact `max-width` property value (like `130px` here), especially when an element is displayed in a single line and thus it mustn't overflow the current line (especially used for mobile). Consequently, using a generic `max-width: 100%` is sufficient to apply an ellipsis to avoid cutting texts (even for `max-width: 100%` you have a CSS helper class: ).

To make this more fluid and adapted, you can add `d-block d-sm-inline-block` into `v-chip` and in this span, you can just add `text-truncate`.

**In addition**, adding an ellipsis without title of the element to display the complete content isn't a good practice at all, since you doesn't give the user the possibility to read the full content.

**BUT**, adding an ellipsis for Mobile view doesn't make sense neither IMO, so please contact the PO to make sure that we shouldn't display the full content in multiple lines instead, especially for **a settings** UI where the user needs to read exactly what was configured.

@rdenarie I let you continue review plz

Reply:
(y)